README.md

```diff
@@ -0,0 +1,91 @@
+Audio Visualizer using DragonBoard410c
+
+# About
+This project aims to demonstrate the various possibilities of using the audio analog header on the DragonBoard410c.
+In this particular project I'll be focusing on the mic input and how that can be used to input audio with Line-In from another audio source like a HiFi system.
+That line-in signal then gets used to drive the NeoPixels connected to an Arduino.
+
+# Table of Contents
+
+- [1) Hardware](#1-hardware)
+  - [1.1) Hardware requirements](#11-hardware-requirements)
+  - [1.2) Hardware setup](#12-hardware-setup)
+- [2) Software](#2-software)
+  - [2.1) DragonBoard410c Setup](#21-dragonboard410c-setup)
+  - [2.1) Arduino Setup](#21-arduino-setup)
+- [3) Setup and Run the Visualizer](#3-setup-and-run)
+
+# 1) Hardware
+
+## 1.1) Hardware requirements
+- [96Boards Dragonboard 410c](https://www.96boards.org/product/dragonboard410c/)
+- 5v 2A USB Power Bank
+- [2x 22uF Bipolar Capacitors](https://in.rsdelivers.com/product/panasonic/ecea1hn220u/panasonic-bipolar-electrolytic-capacitor-22%CE%BCf-50v/7270489)
+- 4x 2.7KOhm Resistors
+- 2x 22KOhm Resistors
+- [1x Arduino Nano](https://store.arduino.cc/usa/arduino-nano)
+- [1x Logic Level Shifter](https://www.amazon.com/Logic-Converter-Bi-Directional-Module-Arduino/dp/B014MC1OAG/ref=sr_1_7?ie=UTF8&qid=1518261841&sr=8-7&keywords=logic+level+converter)
+- [12x NeoPixels in strip or stick](https://www.adafruit.com/category/168)
+- [1x Stereo Audio Socket of choice]
+- Miscellaneous: Quantity as per required:
+ - Connecting wires
+ - Soldering Kit
+ - Screwdriver Set
+ - Double sided tape
+
+## 1.2) Hardware Setup
+- Arduino Setup
+
+| Arduino Nano | Level Shifter HV | <-> | Level Shifter LV | DB410c  LS    | NeoPixels |
```

Comment:
This table is looking neat but I would prefer a schematic diagram which will be better for these kind of multiple connections.

Reply:
I'll add that as well